src/precice/config/ParticipantConfiguration.cpp

```diff
@@ -126,56 +126,46 @@ ParticipantConfiguration::ParticipantConfiguration(
-                               "which is beneficial for a huge mesh and a low number of processors, and a filter "
-                               "\"on-secondary\" strategy, which performs better for a very high number of "
-                               "processors. Both result in the same distribution (if the safety factor is sufficiently large). "
-                               "\"on-primary\" is not supported if you use two-level initialization. "
-                               "For very asymmetric cases, the filter can also be switched off completely (\"no-filter\").")
+                               "A received mesh needs to be decomposed using a mapping-inferred region of interest, if it is not directly-accessed. "
+                               "A geometric filter based on bounding-boxes around the local mesh can speed up this process. "
+                               "`on-primary-rank` is beneficial for a huge mesh and a low number of processors, but is incompatible with two-level initialization. "
+                               "`on-secondary-ranks` performs better for a very high number of processors. "
+                               "Both result in the same distribution if the safety-factor is sufficiently large. "
+                               "`no-filter` may be useful for very asymmetric cases.")
                            .setOptions({VALUE_NO_FILTER, VALUE_FILTER_ON_PRIMARY_RANK, VALUE_FILTER_ON_SECONDARY_RANKS})
                            .setDefaultValue(VALUE_FILTER_ON_SECONDARY_RANKS);
   tagReceiveMesh.addAttribute(attrGeoFilter);
 
-  auto attrDirectAccess = makeXMLAttribute(ATTR_DIRECT_ACCESS, false)
+  tagReceiveMesh.addAttribute(attrFrom);
+  auto attrSafetyFactor = makeXMLAttribute(ATTR_SAFETY_FACTOR, 0.5)
                               .setDocumentation(
-                                  "If a mesh is received from another partipant (see tag <from>), it needs to be"
-                                  "decomposed at the receiving participant. In case a mapping is defined, the "
-                                  "mesh is decomposed according to the local provided mesh associated to the mapping. "
-                                  "In case no mapping has been defined (you want to access "
-                                  "the mesh and related data direct), there is no obvious way on how to decompose the "
-                                  "mesh, since no mesh needs to be provided by the participant. For this purpose, bounding "
-                                  "boxes can be defined (see API function \"setMeshAccessRegion\") and used by selecting "
-                                  "the option direct-access=\"true\".");
-  tagReceiveMesh.addAttribute(attrDirectAccess);
+                                  "The safety-factor of the geometric-filter uniformly scales the bounding box by the given factor. "
```

Comment:
```suggestion
                                  "The safety-factor of the geometric-filter uniformly extends the bounding box by the given relative factor. "
```

Reply:
Added the suggestion from @davidscn 

```suggestion
                                  "The safety factor of the geometric filter uniformly scales the rank-local bounding box by the given factor. "
```